example/onnx/test_super_resolution.py

```diff
@@ -0,0 +1,112 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Testing super_resolution model conversion"""
+from __future__ import absolute_import as _abs
+from __future__ import print_function
+from collections import namedtuple
+import logging
+import numpy as np
+from PIL import Image
+import mxnet as mx
+from mxnet.test_utils import download
+import mxnet.contrib.onnx as onnx_mxnet
+
+# set up logger
+logging.basicConfig()
+LOGGER = logging.getLogger()
+LOGGER.setLevel(logging.INFO)
+
+def download_onnx_model():
+    """Download the onnx model"""
+    model_url = 'https://s3.amazonaws.com/onnx-mxnet/examples/super_resolution.onnx'
+    download(model_url, 'super_resolution.onnx')
+
+def import_onnx():
+    """Import the onnx model into mxnet"""
+    LOGGER.info("Converting onnx format to mxnet's symbol and params...")
+    sym, params = onnx_mxnet.import_model('super_resolution.onnx')
```

Comment:
Is the version (or hash) of this file verified somewhere? To prevent it being updated without us noticing

Reply:
No, it currently does not. I will add the verification. Can you please point me to some examples where this is done, for reference.

Reply:
https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/gluon/data/vision/datasets.py#L152

We're not adopting this behaviour everywhere, but it would be good if we could :)

Reply:
This PR currently has tests for all the operator translations, I plan on making a follow up PR for adding more tests for onnx model translations. I will add a version to the above files and make this change in that follow up PR. Will that be okay?

Reply:
I would prefer tests being introduced at the same time as a new feature. How much is missing?